opentelemetry-sdk/src/opentelemetry/sdk/propagator/b3_format.py

```diff
@@ -57,7 +58,7 @@ def extract(cls, get_from_carrier, carrier):
             elif len(fields) == 4:
                 trace_id, span_id, sampled, _parent_span_id = fields
             else:
-                return trace.INVALID_SPAN_CONTEXT
+                return
```

Comment:
Some kind of error info might be useful to return.

Reply:
in the new system, that would be accomplished by assigning that to the global context object.

I was a bit hesitant to that because chaining propagators is now an option. But for now I can re-add since that's not a feature yet.

Reply:
But for now it could be logged IHMO.